README.md

```diff
@@ -7,7 +7,13 @@
 - CSS preprocessor [SASS](https://sass-lang.com/)
 - Component boilerplate generation tool [generate-react-cli](https://github.com/arminbro/generate-react-cli)
 ---------
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Project Setup
```

Comment:
While we are at it, can you please specify to not use yarn as dependency manager, and strictly go with npm.

Reply:
Or any other dependency manager, for that manager.